packages/devtools_app/lib/devtools.dart

```diff
@@ -9,4 +9,4 @@
 // the constant declaration `const String version =`.
 // If you change the declaration you must also modify the regex in
 // tools/update_version.dart.
-const String version = '2.22.2';
+const String version = '2.22.2-dev.0';
```

Comment:
```suggestion
const String version = '2.22.3-dev.0';
```
I think we need to bump the minor version as well.
Looking at the master log it appears like we already release 2.22.2